analytics/src/main/java/brocoeur/example/analytics/AnalyticsConfigProperties.java

```diff
@@ -0,0 +1,17 @@
+package brocoeur.example.analytics;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Getter
+@Setter
```

Comment:
can it be removed ?

Reply:
No it's compulsory otherwise 

***************************
APPLICATION FAILED TO START
***************************

Description:

Failed to bind properties under 'nerima' to brocoeur.example.nerima.NerimaConfigProperties$$EnhancerBySpringCGLIB$$39ff3dfe:

    Property: nerima.nerima-to-analytics-queue-name
    Value: "NerimaToAnalyticsQueue"
    Origin: class path resource [application.properties] - 4:35
    Reason: java.lang.IllegalStateException: No setter found for property: nerima-to-analytics-queue-name